tests/function_helpers_test.go

```diff
@@ -34,8 +35,8 @@ func deploy(t *testing.T, createRequest *sdk.DeployFunctionSpec) int {
 	return statusCode
 }
 
-func list(t *testing.T, expectedStatusCode int) {
-	functions, err := config.Client.ListFunctions(context.Background(), defaultNamespace)
+func list(t *testing.T, expectedStatusCode int, optional ...interface{}) {
```

Comment:
Can we avoid using this untyped parameter `... interface{}`? What is it doing?

Reply:
Thanks for pointing it out.